examples/kubernetes/orchestrator-template.yaml

```diff
@@ -40,11 +40,20 @@ spec:
             limits:
               memory: "512Mi"
               cpu: "100m"
+          volumeMounts:
+            - mountPath: /orc/conf
+              name: config
         - name: mysql
           image: vitess/orchestrator
           resources:
             limits:
               memory: "512Mi"
               cpu: "100m"
           command: ["mysqld"]
-
+      volumes:
+        - name: config
+          configMap:
+            name: orchestrator-conf
+            items:
```

Comment:
You can leave out the `items` map if you just want the filename to be the same as the key.